src/test/java/org/broadinstitute/hellbender/tools/spark/sv/KmerTest.java

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Created by tsharpe on 1/12/16.
+ */
+public class KmerTest extends BaseTest
```

Comment:
you probably want explicit tests for toString since it's strangely the most complicated method in the class